Pipfile

```diff
@@ -28,7 +28,7 @@ pylint = "*"
 [packages]
 fastapi="==0.65.*"
 elasticsearch2 = ">=2.0.0,<3.0.0"
-elasticsearch = "*"
+elasticsearch = "==7.10.0"
```

Comment:
I fix the version because I still don't explore  why superior version are exploding my RAM..